lib/inspector.coffee

```diff
@@ -69,11 +75,8 @@ class Inspector
             className: 'inspect-message'
             raw: true
 
-    toggleInspectorSize: ->
-        if @inspector?
-            @inspector.toggle()
-
     closeInspector: ->
```

Comment:
On 11/07/16 11:43, Lukas Geiger wrote:
>>      closeInspector: ->
> 
> I don't think we have to have a hydrogen command for hiding the view. This functionality is provided by an button on the view itself.
> 
> If you like to keep |hydrogen:toggle-inspector-size| and |hydrogen:close-inspector| I can add them back.

To be clear, I'm only suggesting to rename the method `@closeInspector` to `@hide`.

As you said, having the command `hydrogen:toggle-inspector` makes |hydrogen:toggle-inspector-size| and |hydrogen:close-inspector| redundant.